workers/unity/Packages/com.improbable.gdk.tools/DownloadCoreSdk.cs

```diff
@@ -25,6 +25,17 @@ internal static class DownloadCoreSdk
 
         private const string DownloadForceMenuItem = "SpatialOS/Download CoreSdk (force)";
 
+        private static readonly List<PluginDirectoryCompatibility> pluginsCompatibilityList = new List<PluginDirectoryCompatibility>
+        {
+            new PluginDirectoryCompatibility("Assets/Plugins/Improbable/Core/Android", false, new List<BuildTarget> { BuildTarget.Android }, null, false),
+            new PluginDirectoryCompatibility("Assets/Plugins/Improbable/Core/iOS", false, new List<BuildTarget> { BuildTarget.iOS }, null, false),
+            new PluginDirectoryCompatibility("Assets/Plugins/Improbable/Core/OSX", false, new List<BuildTarget> { BuildTarget.StandaloneOSX }, null, true),
+            new PluginDirectoryCompatibility("Assets/Plugins/Improbable/Core/Linux", false, new List<BuildTarget> { BuildTarget.StandaloneLinuxUniversal }, null, true),
+            new PluginDirectoryCompatibility("Assets/Plugins/Improbable/Core/Windows", false, new List<BuildTarget> { BuildTarget.StandaloneWindows64 }, null, true),
+            new PluginDirectoryCompatibility("Assets/Plugins/Improbable/Sdk/OSX", true, null, new List<BuildTarget> { BuildTarget.iOS }, true),
```

Comment:
This would require changes to the download program itself to not name the folder OSX as it is every other platform apart from iOS.

Reply:
Yeah, the name is confusing. Shall we rename it to something like "Common" for instance to avoid the confusion?